tools/codegen/pkg/interfacegen/testdata/BUILD

```diff
@@ -0,0 +1,84 @@
+package(default_visibility = ["//tools/codegen/pkg/interfacegen:__pkg__"])
+
+load("@org_pubref_rules_protobuf//protobuf:rules.bzl", "proto_compile")
+load("//tools/codegen:generate.bzl", "mixer_proto_library")
+
+mixer_proto_library(
+    name = "list_template_library",
+    testonly = True,
+    protos = ["ListTemplate.proto"],
```

Comment:
I've filed: https://github.com/istio/mixer/issues/859 to track this outside of this PR (to avoid making this PR gigantic).